src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/severitymatchfilter/Example1.java

```diff
@@ -16,8 +16,8 @@
 
 // xdoc section -- start
 public class Example1 {
-  public void method1(int V1){} // ok, ParameterNameCheck's severity is info
+  public void method1(int V1){} // ok, ParameterName's severity is info
 
-  public void Method2(){} // violation, 'must match pattern'
+  public void Method2(){} // violation, MethodName severity is default it is error
```

Comment:
`MethodName severity is default it is error` => `MethodName's severity is defaulted to error`